utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -688,7 +688,7 @@ function ConvertTo-FormattedJSONParameterObject {
         # [2.4] Syntax:
         # - Everything left of a leftest ':' should be wrapped in quotes (as a parameter name is always a string)
         # - However, we don't want to accidently catch something like "CriticalAddonsOnly=true:NoSchedule"
-        [regex]$pattern = '^\s*\"{0}([0-9a-zA-Z]+):'
+        [regex]$pattern = '^\s*\"{0}([0-9a-zA-Z_]+):'
```

Comment:
Should we merge with #2315 ?

Reply:
Done

Reply:
You merged #2315 with another PR 😆 The idea was to have updates to the readme function in a single location, so that we could regenerate all readmes just once, but at this point nvm. As discussed offline I'll give priority to the API version update PRs

Reply:
I did accidently, right. But merged it in this one after as well. You can see the exact changes also in this PR